software/webapp/src/test/java/org/apache/brooklyn/entity/proxy/TrackingAbstractControllerImpl.java

```diff
@@ -56,7 +55,7 @@ protected void reconfigureService() {
     }
 
     @Override
-    public Class getDriverInterface() {
+    public Class<?> getDriverInterface() {
```

Comment:
For some reason I thought this didn't work (i.e. we couldn't use `Class<?>`). Maybe because the supertype `DriverDependentEntity<D extends EntityDriver>` declares it as `Class<D> getDriverInterface();` and `Class<?>` is not a subtype of `Class<D>`?